tests/infra/ccf.py

```diff
@@ -210,10 +210,11 @@ def start_and_join(self, args):
         cmd = ["rm", "-f"] + glob("member*.pem")
         infra.proc.ccall(*cmd)
 
+        initial_members = list(range(max(1, args.initial_member_count)))
         self.consortium = infra.consortium.Consortium(
-            [0, 1, 2], args.default_curve, self.key_generator
+            initial_members, args.default_curve, self.key_generator
         )
-        self.initial_users = [0, 1, 2]
+        self.initial_users = list(range(max(0, args.initial_user_count)))
```

Comment:
Is there a particular reason why the first member is `1` while the first user is `0`? When the first member/user is registered in CCF, he/she will be given a member/user ID of 0 so I think it is preferable to keep the start index from 0 for now.

Reply:
The index of the first member is 0 already?

Reply:
Good point! We must have at least one member but users are optional at this point.